lib/experimental/fonts/font-library/font-library.php

```diff
@@ -118,3 +118,26 @@ function wp_unregister_font_collection( $collection_id ) {
 );
 
 wp_register_font_collection( $default_font_collection );
+
+/**
+ * Returns relative path to an uploaded font file.
+ *
+ * The path is relative to the current fonts dir.
+ *
+ * @since 6.5.0
+ * @access private
+ *
+ * @param string $path Full path to the file.
+ * @return string Relative path on success, unchanged path on failure.
+ */
+function _wp_relative_fonts_path( $path ) {
```

Comment:
Until this has been shown to be useful elsewhere perhaps better to make this a private function of the class instead?

Reply:
Done.